peps/pep-0725.rst

```diff
@@ -252,11 +305,40 @@ metadata fields. The ``optional-dependencies`` content from ``[external]``
 would need to either reuse ``Provides-Extra`` or require a new
 ``Provides-External-Extra`` field. Neither seems desirable.
 
+The ``dep:`` URLs MUST be converted to PURL identifiers prior to their
+inclusion in the core metadata by following these rules:
+
+- The ``dep`` *scheme* MUST be replaced with ``pkg``.
+- If the *type* is ``compiler`` or ``interface``, it MUST be transformed
+  into a ``virtual/(compiler|interface)`` type.
```

Comment:
That doesn't exist? I'm not sure how this can be handled. The section above says we're omitting a few things, so that may apply here too maybe. That approach may change anyway. Given the recent core metadata bumping, there may be a request to do so for this PEP (I'd prefer not to though, I successfully argued for SBOMs not to need this).

Let's discuss this one.

Reply:
We can also abuse the `Requires-External` metadata and do something hacky like `; False and ...` 😬, but that feels so wrong.

Reply:
The other solution is to drop `virtual` stuff from here, but then their value is diminished and folks might be tempted to write those as `dep:generic/gcc` instead of `dep:virtual/compiler/c`.

Reply:
> 😬, but that feels so wrong.

agreed

> The other solution is to drop `virtual` stuff from here

I'd much prefer to keep it as you have it now.